system/lib/libc/musl/arch/emscripten/bits/syscall.h

```diff
@@ -53,7 +53,6 @@
 #define SYS_stat64		__syscall_stat64
 #define SYS_lstat64		__syscall_lstat64
 #define SYS_fstat64		__syscall_fstat64
-#define SYS_lchown32		__syscall_lchown32
```

Comment:
Can you also remove these from `system/lib/libc/musl/arch/emscripten/syscall_arch.h`?